lib/lago/__init__.py

```diff
@@ -620,8 +643,8 @@ def _ova_to_spec(self, filename):
 
         if image_file is not None:
             disk_meta = {"root-partition": "/dev/sda1"}
-
-            disk_spec = [{"type": "file",
+            disk_spec = [{"type": "template",
+                          "template_type": "file",
```

Comment:
This is a temporary file in prefix.
Exactly the same as files inside images directory
its removed together with prefix